src/coreclr/src/pal/src/init/pal.cpp

```diff
@@ -359,18 +359,34 @@ Initialize(
         gPID = getpid();
         gSID = getsid(gPID);
 
+        // Initialize the TLS lookaside cache
```

Comment:
Nit: This comment is wrong. We got rid of the TLS lookaside cache a long time ago.

Reply:
Fixed